tests/foreman/cli/test_myaccount.py

```diff
@@ -165,23 +159,64 @@ def test_negative_update_email(self):
         for email in invalid_emails_list():
             with self.subTest(email):
                 with self.assertRaises(CLIReturnCodeError):
-                    User.update({
-                        'login': self.user['login'],
-                        'mail': email,
-                    })
+                    self.update_user(
+                        {'login': self.user['login'], 'mail': email}
+                    )
 
-    @stubbed()
     @tier1
-    def test_negative_update_password_invalid(self):
-        """Update My Account with invalid Password/Verify fields
+    def test_negative_update_locale(self):
+        """Update My Account with invalid Locale
 
-        @id: f9230699-fb8e-45d6-a0c2-abb8b751304d
-
-        @Steps:
-        1. Update Current user with all variations of Password/Verify fields
-        in [2]
+        @id: 6f63c9b4-80e6-11e6-8ea1-68f72889dc7f
 
         @Assert: User is not updated. Appropriate error shown.
+        """
+        with self.assertRaises(CLIReturnCodeError):
+            self.update_user({'locale': 'invalid'})
 
-        @caseautomation: notautomated
+
+class MyAccountEphemeralUserTestCase(CLITestCase):
+    """Implements My Account functionality tests in CLI for test that need
+    to use ephemeral user. E.g. one user is created for each test so it can
+    be whatever field updated"""
+
+    def setUp(self):
+        """Create a new user for each myaccount tests to not impact the default
+        user.
         """
+        super(MyAccountEphemeralUserTestCase, self).setUp()
+        self.login = gen_string('alphanumeric', 30)
+        self.password = gen_string('alphanumeric', 30)
+        self.user = make_user({
+            'login': self.login,
+            'password': self.password,
+            'admin': '1',
+        })
+
+    def tearDown(self):
+        """Delete user created for tests"""
+        super(MyAccountEphemeralUserTestCase, self).tearDownClass()
```

Comment:
Just pushed some changes fixing it and making code more clear. See @oshtaier discussion as well